docs/howtos/DataPlane Generation - DPG/10longRunningOperations.mdx

```diff
@@ -0,0 +1,188 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Long-Running Operations
```

Comment:
Some services may have requirement to expose polling operation, should we have an example for that case?

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/apicenter/ApiCenter.DataApi/resources/ApiDefinition.tsp#L27

Reply:
I am currently not sure whether SDK reviewers want to have this API in SDK API.

Does JS have some way to get the "id" easily and allow customer call this GET API with this "id"?

In Java, it is not easy for customer to get the "id"... (in standard LRO, customer can read the "id" from the poll response, and use that -- but this is not intuitive)

Reply:
I am asking because I notice when I searched the `LongRunningResource` I found almost operations would define their polling operation e.g @pollingOperation(OperationStatuses.get) but not every exposed it in client.tsp.

So I was wondering if this is common case or should their polling operation be removed?

Reply:
It depends. In my opinion, if they use the standard LRO here, `@pollingOperation` should be removed.

But if they use their customized LRO, they may need it.

Reply:
Got it and i am okay not adding customized polling here if this is not common case.

Reply:
I am checking in channel about what is the best practice.
https://teams.microsoft.com/l/message/19:16f8f1ed70d044139be153707bdaee0e@thread.skype/1723790726909?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1723790726909&teamName=Azure%20SDK&channelName=TypeSpec%20Engineering&createdTime=1723790726909

Reply:
Mark's reply says they required the pollingOperation. I will add a `getOperationStatus` but comment that SDK may not want to include it.

I remember a long discussion months before, about whether emitter include the op in `@pollingOperation`. I remember the decision is to handle it like any other operation.
But there is way to exclude it. One be client.tsp `@client` omit it. Another to define it in another namespace (I didn't see any service did this, though).